Src/BootCamp.Chapter/Check.cs

```diff
@@ -0,0 +1,38 @@
+﻿// This file is for hunting bugs only.
+// Completing homework 3 before looking at this is HIGHLY recommended.
+// Try to look at the code in GitHub first. Try to find the mistakes that
+// were made without help or tools first.
+// After that try to find every single thing that seems off.
+// Have fun! :)
+namespace BootCamp.Chap
+{
+    class Checks // class is now private so the functions could not be used outside the class and all functions must be static so you do not have to initialize a class to use them.
+    {
+        public string PromptString(String Message) // this must be string because you want the type string not the class string.
+        {
+           // No using statement for System so Console is unknown.
+           // No call to the own function so it is not testable. 
+            return Console.ReadLine(); 
+        }
+
+        public int PromptInt(String message) // this must be string because you want the type string not the class string.
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make 'PromptInt' a static method.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4885581)